packages/terra-time-input/src/terra-dev-site/doc/time-input/ChangeLog.doc.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Lets update this name to be `....time-input/ChangeLog.b.doc.jsx` and the Index page to be `...dev-site/doc/time-input/TimeInput.a,doc.jsx` such that the Index page is the first item in the list.

Currently appears as:
![screen shot 2018-06-25 at 12 50 56 pm](https://user-images.githubusercontent.com/14099737/41866664-7455addc-7876-11e8-9bd6-a7c9d7fd8710.png)

But expect:

![screen shot 2018-06-25 at 12 51 42 pm](https://user-images.githubusercontent.com/14099737/41866693-88cbf87a-7876-11e8-849d-b2b8424dece6.png)

Reply:
If the secondary sort can sort numerically that might make a little more sense.

Component.1.jsx
Changelog.2.jsx
UpgradeGuide.3.jsx